### PR TITLE
Use yaml formatter to format ansible files

### PR DESCRIPTION
**Pull request by robsonpeixoto**
The Ansible[1] extension define a new language for yaml files that describe
ansible playbooks and because of that is not possible use the prettier
extension with ansible files.

1 - https://marketplace.visualstudio.com/items?itemName=vscoss.vscode-ansible

- [x] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/pr-XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Comment by robsonpeixoto** [comment]
@evilebottnawi  done in f02c9f7b0684304482804dd62fd447175e4cb095

File: .cspell.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "version": "0.1",
+  "words": [
+    "ansible"
+  ]
+}
```
Comment (fisker) [inline .cspell.json:6]: We already has https://github.com/prettier/prettier/blob/master/cspell.json

Comment (robsonpeixoto) [reply .cspell.json]: Thanks. Fixed!

Comment (fisker) [reply .cspell.json]: Sorry, but you need fix indention, and remove the final newline. https://github.com/prettier/prettier/pull/6684#discussion_r336745585

Comment (robsonpeixoto) [reply .cspell.json]: Done @fisker

**Comment by robsonpeixoto** [comment]
Finally all green, @fisker and @evilebottnawi

**Comment by fisker** [comment]
Looks like the test  added is  just a normal yaml file, is there any difference between `yaml` and `ansible`? Or  any special extension?

**Comment by robsonpeixoto** [comment]
No @fisker, is just a ordinary yaml file. I opened this PR because of this: https://github.com/prettier/prettier-vscode/issues/1259

**Comment by robsonpeixoto** [comment]
Thanks =D

**Pull request merged**